src/platform/unix.rs

```diff
@@ -406,3 +338,94 @@ impl MonitorHandleExtUnix for MonitorHandle {
         self.inner.native_identifier()
     }
 }
+
+/// Wrapper for implementing SCTK's theme trait.
+pub struct WaylandTheme<T: Theme>(T);
```

Comment:
I believe this should be private. Or `pub(crate)`, if necessary.

Reply:
Yeah, there's absolutely no reason why this would be public, muscle memory just kicked in for me.